platform.md

```diff
@@ -327,126 +395,63 @@ The lifecycle:
-| `200-299` | Analysis-specific lifecycle errors
-
-- The lifecycle MUST write [analysis metadata](#analyzedtoml-toml) to `<analyzed>` if `<image>` is accessible.
-- **If** `<skip-layers>` is `true` the lifecycle MUST NOT perform layer analysis.
-- **Else** the lifecycle MUST analyze any app image layers or cached layers created by any buildpack present in the provided `<group>`.
-
-##### Layer analysis
-When analyzing a given layer the lifecycle SHALL:
-- **If** `build=true`, `cache=false`:
-    - Do nothing
-- **Else if** `launch=true`:
-    - Write layer metadata read from the analyzed image to `<layers>/<buildpack-id>/<layer-name>.toml`
-    - Write the layer diffID from the analyzed image to `<layers>/<buildpack-id>/<layer-name>.sha`
-- **Else if** `cache=true`:
-    - Write layer metadata read from the cache to `<layers>/<buildpack-id>/<layer-name>.toml`
-    - Write the layer diffID from the cache to `<layers>/<buildpack-id>/<layer-name>.sha`
-
-#### `restorer`
-Usage:
-```
-/cnb/lifecycle/restorer \
-  [-cache-dir <cache-dir>]
-  [-cache-image <cache-image>]
-  [-gid <gid>] \
-  [-group <group>] \
-  [-layers <layers>] \
-  [-log-level <log-level>] \
-  [-uid <uid>]
-```
+| `40-49` | Restoration-specific lifecycle errors
 
-##### Inputs
-| Input          | Environment Variable  | Default Value         | Description
-|----------------|-----------------------|-----------------------|----------------------
-| `<cache-dir>`  | `CNB_CACHE_DIR`       |                       | Path to a cache directory
-| `<cache-image>`| `CNB_CACHE_IMAGE`     |                       | Reference to a cache image in an OCI image registry
-| `<gid>`        | `CNB_GROUP_ID`        |                       | Primary GID of the stack `User`
-| `<group>`      | `CNB_GROUP_PATH`      | `<layers>/group.toml` | Path to group definition (see [`group.toml`](#grouptoml-toml))
-| `<layers>`     | `CNB_LAYERS_DIR`      | `/layers`             | Path to layers directory
-| `<log-level>`  | `CNB_LOG_LEVEL`       | `info`                | Log Level
-| `<uid>`        | `CNB_USER_ID`         |                       | UID of the stack `User`
-| `<layers>/<buidpack-id>/<layer>.sha`  ||                       | Files containing the diffID of each analyzed layer
-| `<layers>/<buidpack-id>/<layer>.toml` ||                       | Files containing the layer content metadata of each analyzed layer (see data format in [Buildpack Interface Specification](buildpack.md))
+- For each buildpack in `<group>`, if persistent metadata for that buildpack exists in the analysis metadata, lifecycle MUST write a toml representation of the persistent metadata to `<layers>/<buildpack-id>/store.toml`
+- **If** `<skip-layers>` is `true` the lifecycle MUST NOT perform layer restoration.
+- **Else** the lifecycle MUST perform [layer restoration](#layer-restoration) for any app image layers or cached layers created by any buildpack present in the provided `<group>`.
```

Comment:
Does this apply to app image layers? `#layer-restoration` only talks about the cache.